minicluster/src/main/java/tachyon/master/AbstractLocalTachyonCluster.java

```diff
@@ -307,19 +304,6 @@ protected void stopUFS() throws Exception {
   }
 
   /**
-   * Reset the {@link LoginUser}. This is called when the cluster is stopped.
-   *
-   * @throws Exception when the operation fails
-   */
-  private void resetLoginUser() throws Exception {
```

Comment:
No, I most likely made a mistake during the merge conflict resolution, I've added it back, thanks.